### ISSUE TITLE
Move UI icons to a UFO file in the resources directory

### ISSUE THREAD

**Issue by eliheuer** [linebender/runebender#248]
An idea from a discussion in this pull request: https://github.com/linebender/runebender/pull/247

Would it be possible to load icons from a UFO file in the `resources` directory? Currently, I am drawing the icons in Runebender and pasting them into the `toolbar.rs` file (see the screenshot below). 

<img width="1920" alt="rb-icons" src="https://user-images.githubusercontent.com/5162664/117418193-fc57ae00-aecf-11eb-830d-8ddab9bba22e.png">

I have a git repo for the icon UFOs I have worked with so far [here](https://github.com/eliheuer/runebender-icons/tree/main/ufo-icon-source/toolsbar-icons.ufo). It might be nice to just add a UFO like this to the resources directory so an external repo isn't needed to track these files.

**Comment by alerque** [comment]
Definitely the right move, the actual icon source that a developer could tweak without starting from scratch should not be in a separate repository ;-)

Since toolbar icons aren't a full fledged font and don't need kerning or OpenType features or anything, perhaps storing a full UFA isn't necessary and just bare `.glif` files would be enough.

**Comment by ctrlcctrlv** [comment]
Being able to edit detached (unparented) .glif files is a must if you ask me, but I'm trying not to comment here too much given that I, you know, am the BD of a competing project. ;-)

**Comment by eliheuer** [comment]
There is some further discussion about this going on here: https://github.com/linebender/runebender/issues/248

I haven't tried it myself, but @ctrlcctrlv's idea about loading just `.glif` files in the editor seems like it would work well for this.

**Comment by cmyr** [comment]
I agree this makes total sense. I also think individual `glif` files is a nice idea; I've been thinking it would be nice to try and keep the coupling between runebender and the UFO format a little bit loose, and supporting some basic non-UFO cases is a good way to force that, and individual `glif` files would be a very simple way to do that, maybe?
